Content.Client/Physics/Controllers/MoverController.cs

```diff
@@ -128,7 +129,11 @@ private void HandleClientsideMovement(EntityUid player, float frameTime)
             }
 
             // Server-side should just be handled on its own so we'll just do this shizznit
-            HandleMobMovement(player, mover, body, xformMover, frameTime, xformQuery, moverQuery, relayTargetQuery);
+            if (body.Predict && !UseMobMovement(player))
+            {
+                var speedMod = CompOrNull<MovementSpeedModifierComponent>(player);
+                HandleMobMovement(player, mover, body, xformMover, frameTime, xformQuery, moverQuery, relayTargetQuery, speedMod);
+            }
```

Comment:
Pointless changes (Predict is already set above and mobmovement is checked internally).

Reply:
predict is useless yes, but if i don't check mob movement i get assertion error here (beginning of HandleMobMovement)